pkg/fission-cli/cmd/spec/command.go

```diff
@@ -60,13 +60,22 @@ func Commands() *cobra.Command {
 		Optional: []flag.Flag{flag.SpecDir},
 	})
 
+	listCmd := &cobra.Command{
+		Use:   "list",
+		Short: "List all Fission resources in the application specification",
+		RunE:  wrapper.Wrapper(List),
+	}
+	wrapper.SetFlags(listCmd, flag.FlagSet{
+		Optional: []flag.Flag{flag.SpecDir},
```

Comment:
How about we add a flag that allows a user to specify the deployID directly instead of moving into spec dir everytime?